internal/cmd/base/servers.go

```diff
@@ -711,21 +717,16 @@ func (s *Server) SetStatusGracePeriodDuration(value time.Duration) {
 		v := os.Getenv(statusGracePeriodEnvVar)
 		n, err := strconv.Atoi(v)
 		if err != nil {
-			s.Logger.Error(fmt.Sprintf("could not read setting for %s", statusGracePeriodEnvVar),
-				"err", err,
-				"value", v,
-			)
+			event.WriteError(ctx, op, err, event.WithInfoMsg(fmt.Sprintf("could not read setting for %s", statusGracePeriodEnvVar), "value", v))
```

Comment:
Maybe `"WithInfoMsg("could not read status grace period setting", "envvar", statusGracePeriodEnvVar, "value", v)`